### DISCUSSION TITLE
Branch Name Argument?

### DISCUSSION THREAD

**Discussion by mcdonnell-john** [alirezanet/Husky.Net#97] [Q&A] [answered]
Hi,

I just came across this project as I was looking to introduce git hooks into our project, specifically a pre-push hook to verify that a branch name conforms to our standard naming convention.  I can see there's args that can be passed into each hook, just wondering if there is a prebuilt arg for branch name, or how I can get it?

**Comment by alirezanet** [comment, marked as answer]
Hi @mcdonnell-john,
There is no built-in branch name variable but you can define your [custom one.](https://alirezanet.github.io/Husky.Net/guide/task-configuration.html#custom-variables)

to get the active branch name you can use the below command:
```
git symbolic-ref --short -q HEAD
```
